pepper-apis/dsm-core/src/main/resources/liquibase/rgp/PEPPER-833_rgp_kit_display_name.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-833_rgp_kit_type_display_name" author="pegah">
+        <addColumn tableName="kit_type">
+            <column name="kit_display_name" type="VARCHAR(500)"></column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="PEPPER-833_rgp_blood" author="pegah">
+        <insert tableName="kit_type">
+            <column name="kit_type_name" value="BLOOD"></column>
+            <column name="bsp_material_type" value="Whole Blood Streck Cell-Free Preserved"></column>
+            <column name="bsp_receptacle_type" value="Vacutainer Cell-Free DNA Tube Camo-Top [10mL]"></column>
+            <column name="requires_insert_in_kit_tracking" value="1"></column>
+            <column name="kit_display_name" value="BLOOD &amp; RNA"></column>
+        </insert>
+    </changeSet>
+    <changeSet id="PEPPER-833_update_rgp_blood_subkit" author="pegah">
+        <update tableName="sub_kits_settings">
+            <column name="kit_type_id" valueComputed="(SELECT kit_type_id FROM kit_type WHERE kit_display_name = 'BLOOD &amp; RNA' )"></column>
+            <where>
```

Comment:
This feels odd. We are using the display name to figure out the kit type ID? Developers might feel they can change a field with the term `display` in it. 
Is there a reason we do not have a unique key in `kit_type`?
What's the overall goal here?
What is the `sub_kits_settings` table used for?

Reply:
Yes, we kinda have to. The display name is the only thing making this field different than the others. Also kit types unfortunately are not unique, and my guess is that there could be many kits called blood but having different meanings for each study or different metadata. You can look at dev database to see it. 
I have added the general explanation of subkits to a new doc and will send it to you, but basically `sub_kits_settings` is the settings for when a package of kits contains multiple kit types and tubes in it. For example in this ticket RGP wants to send one package to each participant with 2 tubes, one for blood and one for RNA extraction